PENDING.md

```diff
@@ -18,6 +18,37 @@ BREAKING CHANGES
     utilize a validator's operator address must now use the new Bech32 prefix,
     `cosmosvaloper`.
     * [cli] [\#2190](https://github.com/cosmos/cosmos-sdk/issues/2190) `gaiacli init --gen-txs` is now `gaiacli init --with-txs` to reduce confusion
+    * [\#1184](https://github.com/cosmos/cosmos-sdk/issues/1184) Subcommands reorganisation, a comprehensive list of renamings follow:
+      - `tendermint validator-set` -> `query tendermint-validator-set`
```

Comment:
Let's not put the whole list in the pending file, linking to the issue is enough.

Reply:
Agreed, amended